frontend/src/components/Form.js

```diff
@@ -82,9 +80,10 @@ function Form() {
 
   function addStep() {
     // Generate a dynamic number of inputs
-    numSteps++;
```

Comment:
what does this numStep do, and do we use the value of this variable somewhere else?

Reply:
Yeah this was a bug, it was supposed to keep track of the steps for the recipe but it wasn't saving state. Now we just check how many inputs with a class name "step" there are in the form